docs/logs/getting-started/README.md

```diff
@@ -17,12 +17,17 @@ You should see the following output:
 Hello World!
 ```
 
+Install the Microsoft.Extensions.Logging package:
+
+```sh
+dotnet add package Microsoft.Extensions.Logging -v 5.0.0-rc.2.20475.5
```

Comment:
lets move to 5.0.0 version

Reply:
I guess we can remove the `-v xxx` part entirely? It should give the latest stable version.

Reply:
If we don't specify the version it currently installs 5.0.0 but that leads to a conflict:
`error NU1107: Version conflict detected for System.Diagnostics.DiagnosticSource. Install/reference System.Diagnostics.DiagnosticSource 5.0.0 directly to project getting-started-logs to resolve this issue.`
`error NU1107:  getting-started-logs -> Microsoft.Extensions.Logging 5.0.0 -> System.Diagnostics.DiagnosticSource (>= 5.0.0)`
`error NU1107:  getting-started-logs -> OpenTelemetry.Exporter.Console 0.8.0-beta.1 -> OpenTelemetry 0.8.0-beta.1 -> OpenTelemetry.Api 0.8.0-beta.1 -> System.Diagnostics.DiagnosticSource (= 5.0.0-rc.2.20475.5).`

Reply:
@utpilla , i think that is a different issue...we could use without the version verb and fix those packages that have different numbers. to facilitate, i think exists a tool called https://github.com/dotnet-outdated/dotnet-outdated, which can update everything